e2e/test/scenarios/collections/collection-items-listing.cy.spec.js

```diff
@@ -87,127 +87,124 @@ describe("scenarios > collection items listing", () => {
       archiveAll();
     });
 
-    it(
-      "should allow to sort unpinned items by columns asc and desc",
-      { tags: "@flaky" },
-      () => {
-        ["A", "B", "C"].forEach((letter, i) => {
-          cy.createDashboard({
-            name: `${letter} Dashboard`,
-            collection_position: null,
-          });
-
-          // Signing in as a different users, so we have different names in "Last edited by"
-          // In that way we can test sorting by this column correctly
-          cy.signIn("normal");
-
-          cy.createQuestion({
-            name: `${letter} Question`,
-            collection_position: null,
-            query: TEST_QUESTION_QUERY,
-          });
+    it("should allow to sort unpinned items by columns asc and desc", () => {
+      ["A", "B", "C"].forEach((letter, i) => {
+        cy.createDashboard({
+          name: `${letter} Dashboard`,
+          collection_position: null,
         });
 
-        visitRootCollection();
+        // Signing in as a different users, so we have different names in "Last edited by"
+        // In that way we can test sorting by this column correctly
+        cy.signIn("normal");
 
-        getAllCollectionItemNames().then(({ actualNames, sortedNames }) => {
-          expect(actualNames, "sorted alphabetically by default").to.deep.equal(
-            sortedNames,
-          );
+        cy.createQuestion({
+          name: `${letter} Question`,
+          collection_position: null,
+          query: TEST_QUESTION_QUERY,
         });
+      });
 
-        toggleSortingFor(/Name/i);
-        cy.wait("@getCollectionItems");
+      visitRootCollection();
+      // We're waiting for the loading spinner to disappear from the main sidebar.
+      // Otherwise, this causes the page re-render and the flaky test.
+      cy.findByTestId("main-navbar-root").get("circle").should("not.exist");
```

Comment:
This is an actual fix.
The rest is the whitespace diff after cd56ce7903de60dbcc0ed4c365a5a7261927ad85